docs/ENV.md

```diff
@@ -70,6 +70,9 @@ AWS secret access key.
 #### `AWS_REGION`
 Region of the AWS IAM user.
 
+#### `AWS_STORAGE_BUCKET_NAME`
```

Comment:
have these been made different for staging and prod?

Reply:
no not yet, will do

Reply:
done